src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -14324,6 +14325,123 @@ public void TestDuplicateAdditionalFiles(string additionalFilePath1, string addi
         [InlineData("abc/a.txt", "./../ABC/a.txt", 2)]
         public void TestDuplicateAdditionalFiles_Linux(string additionalFilePath1, string additionalFilePath2, int expectedCount) => TestDuplicateAdditionalFiles(additionalFilePath1, additionalFilePath2, expectedCount);
 
+
```

Comment:
How does this behavior interact with `[CallerFilePath]` optional arguments? Now that the source file has a full path would expect some interaction with this feature.